showpass/Showpass-E2E-tests
# TA-29-Profile - Change user's password and ensure it can be used to log in.

**Description:**
End users should be able to update their password and use these new credentials to log into the portal. This user story will be verifying that it is still fully functional.

Steps:
- Navigate to https://beta.showpass.com/account/profile/change-password/
- Provide old and the new passwords
- Click ‘Save’
- Sign out
- Log back into the portal using the new password

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-29

**How to test:**
Run the following spec and ensure it passes
cypress/e2e/user-settings/destructive-profile-updates.cy.js

**Test execution results:**

<img width="1718" alt="TA-29-passed" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/5eb0f111-87c2-4caa-bb57-a15d2794ce06">
